app/scripts/services/os-admin/index.js

```diff
@@ -91,7 +91,9 @@ function getDataPackages(authToken, userid) {
   if (authToken) {
     url += '&jwt=' + encodeURIComponent(authToken);
   }
-  url += '&package.owner=' + encodeURIComponent(JSON.stringify(userid));
+  if (userid) {
```

Comment:
In short - we **should** always have user id. It's rather a way to ensure that code will not fail if **accidentally** there will be no user id.